Makefile

```diff
@@ -393,7 +393,6 @@ androidupdate: all
 		-xr!*clipboard$ \
 		-xr!*data/dict$ \
 		-xr!*data/tessdata$ \
-		-xr!*history$ \
```

Comment:
This comes across as counter to this PR, or was it previously excluded by accident?